spec/models/game_spec.rb

```diff
@@ -165,6 +166,29 @@
       expect(game.checkmate?(black_king.is_white)).to eq(true)
     end
 
+    it 'should return true if king is in checkmate' do
+      game = FactoryGirl.create(:game)
+      black_king = FactoryGirl.create(:king, is_white: false, game: game, x_position: 3, y_position: 4)
+      FactoryGirl.create(:pawn, is_white: false, game: game, x_position: 2, y_position: 5)
+      FactoryGirl.create(:pawn, is_white: false, game: game, x_position: 1, y_position: 6)
+      FactoryGirl.create(:pawn, is_white: false, game: game, x_position: 8, y_position: 7)
+      FactoryGirl.create(:rook, is_white: false, game: game, x_position: 8, y_position: 8)
+      FactoryGirl.create(:knight, is_white: false, game: game, x_position: 7, y_position: 8)
+      FactoryGirl.create(:queen, is_white: true, game: game, x_position: 3, y_position: 7)
+      FactoryGirl.create(:bishop, is_white: true, game: game, x_position: 2, y_position: 6)
+      FactoryGirl.create(:rook, is_white: true, game: game, x_position: 5, y_position: 6)
+      FactoryGirl.create(:pawn, is_white: true, game: game, x_position: 1, y_position: 4)
+      FactoryGirl.create(:pawn, is_white: false, game: game, x_position: 2, y_position: 4)
```

Comment:
As mentioned earlier, the test fails if this pawn is set as black, which is what I had it set as in the example where I found the issue.

Reply:
I'm sorry, I meant if pawn is WHITE, so that the black king can capture it and move to that square (2 , 4), which is under attack by another white pawn.

Reply:
Okay, I'm changing it back to white...